var/spack/repos/builtin/packages/py-radical-utils/package.py

```diff
@@ -7,14 +7,28 @@
 
 
 class PyRadicalUtils(PythonPackage):
-    """Shared code and tools for various RADICAL Projects"""
+    """RADICAL-Utils contains shared code and tools for various
+    RADICAL-Cybertools packages."""
 
-    homepage = "http://radical.rutgers.edu"
-    pypi = "radical.utils/radical.utils-0.45.tar.gz"
+    homepage = 'https://radical-cybertools.github.io'
+    git      = 'https://github.com/radical-cybertools/radical.utils.git'
+    pypi     = 'radical.utils/radical.utils-1.6.7.tar.gz'
 
-    version('0.45', sha256='1333cff1a69532e51d4484fbac3fad6b172d415d2055a3141117c7cf8bdee6c5')
-    version('0.41.1', sha256='582900e0434f49b69885a89bc65dc787362756e1014d52a4afac0bb61bcaa3ce')
```

Comment:
We don't usually remove older versions without first deprecating them. Is there a reason you need to remove these older versions?

Reply:
these versions are completely out of sync and could mislead

Reply:
I would still deprecate them before removing, especially since these were previously the default versions used by everyone who installed this package with Spack.

Reply:
will do, thanks!